Resources/Prototypes/Entities/Objects/Specific/Hydroponics/plant_analyzer.yml

```
@@ -0,0 +1,53 @@
+- type: entity
+  id: PlantAnalyzer
+  parent: BaseItem
+  name: plant analyzer
+  description: A scanner used to evaluate a plant's various areas of growth, genetic traits and chemicals.
+  components:
+  - type: Sprite
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)